test/functional/p2p_orphan_handling.py

```diff
@@ -357,9 +361,11 @@ def test_orphan_of_orphan(self):
 
         # The node should put the orphan into the orphanage and request missing_parent, skipping
         # missing_parent_orphan because it already has it in the orphanage.
+        peer.clear_getdata()
         self.relay_transaction(peer, orphan["tx"])
         self.nodes[0].bumpmocktime(NONPREF_PEER_TX_DELAY + TXID_RELAY_DELAY)
         peer.wait_for_parent_requests([int(missing_parent["txid"], 16)])
+        peer.assert_never_requested(int(missing_parent_orphan["txid"], 16))
```

Comment:
`wait_for_parent_requests` already checks this, since it requires that the list of getdatas match exactly (nothing additional allowed)?

Reply:
hmm yeah this doesn't add anything *unless it was requested in a separate `getdata` which this check wouldn't ensure anyways.